app/models/user.rb

```diff
@@ -4,4 +4,7 @@ class User < ApplicationRecord
   has_one_attached :profile_picture
   validates :full_name, :gender, :country, :city, :username, presence: true
   validates :username, uniqueness: true
+  validates :password, presence: true
```

Comment:
maybe we can do this => `validates :password, :password_confirmation, presence: true`